web3_proxy/src/frontend/users/stats.rs

```diff
@@ -53,8 +53,22 @@ pub async fn user_revert_logs_get(
         .await
         .web3_context("failed loading user's key")?;
 
+    // Also add rpc keys for which this user has access
+    let shared_rpc_keys = secondary_user::Entity::find()
+        .filter(secondary_user::Column::UserId.eq(user.id))
+        .all(db_replica.conn())
+        .await?
+        .into_iter()
+        .map(|x| x.rpc_secret_key_id);
+
+    // We shouldn't need to be deduped, bcs the set of shared keys is distinct from the user's keys,
+    // the database also handles deduplication bcs it's a projection operation
     // TODO: only select the ids
-    let uks: Vec<_> = uks.into_iter().map(|x| x.id).collect();
+    let uks: Vec<_> = uks
+        .into_iter()
+        .map(|x| x.id)
```

Comment:
this should attach a "owner" role to this id. that way it can be zipped up with the shared_rpc_keys